rpc/src/main/scala/server/implicits.scala

```diff
@@ -29,8 +29,8 @@ trait Syntax {
 
   final class ServerOps(server: FreeS[GrpcServer.Op, Unit]) {
 
-    def bootstrapM[M[_]](implicit MM: Monad[M], handler: GrpcServer.Op ~> M): M[Unit] =
-      server.interpret[M]
+    def bootstrapM[F[_]](implicit M: Monad[F], handler: GrpcServer.Op ~> F): F[Unit] =
```

Comment:
Regarding the naming conventions mentioned earlier. should this just be `M[_]: Monad` instead of an implicit arg that is not in use in the method body?